docs/self-managed/platform-deployment/helm-kubernetes/platforms/amazon-eks/dual-region.md

```diff
@@ -0,0 +1,534 @@
+        }
+    }
+```
+
+  </summary>
+</details>
+
+2. Repeat **Step 1** for the Paris cluster to configure the CoreDNS of Paris to redirect any DNS requests to the London DNS resolver for your chosen namespace, like `camunda-london`.
+
+3. Check that CoreDNS has reloaded for the changes to take effect before continuing. Make sure it contains `Reloading complete`:
+
+```bash
+kubectl --context $CLUSTER_0 logs -f deployment/coredns -n kube-system
+kubectl --context $CLUSTER_1 logs -f deployment/coredns -n kube-system
+```
+
+</TabItem>
+<TabItem value="test-dns-chaining" label="(optional) Test DNS chaining">
+
+A setup to test that the DNS chaining is working by using nginx pods and services to ping each other.
+
+1. Deploy the following nginx pod and service to each cluster within the `camunda-london` and `camunda-paris` namespaces:
+
+```bash
+kubectl --context $CLUSTER_0 apply -f https://github.com/camunda/c8-multi-region/blob/main/test/resources/aws/2-region/kubernetes/nginx.yml -n $CAMUNDA_NAMESPACE_0
+kubectl --context $CLUSTER_1 apply -f https://github.com/camunda/c8-multi-region/blob/main/test/resources/aws/2-region/kubernetes/nginx.yml -n $CAMUNDA_NAMESPACE_1
+```
+
+2. Try to curl each other from within the other pod:
+
+```bash
+kubectl --context $CLUSTER_0 exec -n $CAMUNDA_NAMESPACE_0 -it sample-nginx -- curl http://sample-nginx.sample-nginx-peer.$CAMUNDA_NAMESPACE_0.svc.cluster.local
+
+kubectl --context $CLUSTER_1 exec -n $CAMUNDA_NAMESPACE_1 -it sample-nginx -- curl http://sample-nginx.sample-nginx-peer.$CAMUNDA_NAMESPACE_1.svc.cluster.local
+```
+
+3. After a successful retrieval of the remote nginx start page, you can remove this temporary setup.
+
+```bash
+kubectl --context $CLUSTER_0 delete -f https://github.com/camunda/c8-multi-region/blob/main/test/resources/aws/2-region/kubernetes/nginx.yml -n $CAMUNDA_NAMESPACE_0
+kubectl --context $CLUSTER_1 delete -f https://github.com/camunda/c8-multi-region/blob/main/test/resources/aws/2-region/kubernetes/nginx.yml -n $CAMUNDA_NAMESPACE_1
+```
+
+</TabItem>
+</Tabs>
+
+## Deploy Camunda 8 to the clusters
+
+<Tabs queryString="deploy">
+<TabItem value="elastic-secret" label="Create the secret for Elasticsearch">
+
+Elasticsearch will use S3 as a backup and restore bucket. For this, configure a Kubernetes secret to not expose those in cleartext.
+
+You can pull the data from Terraform since you exposed those via the `output.tf`.
+
+1. From the Terraform code location `test/resources/aws/2-region/terraform`, execute the following to export the access keys to environment variables. This will allow an easier creation of the Kubernetes secret via the command line:
+
+```bash
+export ACCESS_KEY=$(terraform output -raw s3_aws_access_key_id)
+export SECRET_ACCESS_KEY=$(terraform output -raw s3_aws_secret_access_key)
+```
+
+2. Pre-create the Camunda namespaces in each cluster. This is required to have the secrets available for Elasticsearch before installing the Helm chart.
+
+```bash
```

Comment:
Does it make sense to move this whole step into a small shell script that creates the secrets?